app/_hub/kong-inc/acl/index.md

```diff
@@ -93,16 +85,20 @@ params:
       description: |
         Flag that if enabled (`true`), prevents the `X-Consumer-Groups` header to be sent in the request to the Upstream service.
   extra: |
-    Note that the `allow` and `deny` models are mutually exclusive in their usage, as they provide complimentary approaches. That is, you cannot configure an ACL with both `allow` and `deny` configurations. An ACL with an `allow` provides a positive security model, in which the configured groups are allowed access to the resources, and all others are inherently rejected. By contrast, a `deny` configuration provides a negative security model, in which certain groups are explicitly denied access to the resource (and all others are inherently allowed).
+    Note that you cannot configure an ACL with both `allow` and `deny` configurations. An ACL with an `allow` provides a positive security model, in which the configured groups are allowed access to the resources, and all others are inherently rejected. By contrast, a `deny` configuration provides a negative security model, in which certain groups are explicitly denied access to the resource (and all others are  allowed).
 ---
 
 ### Usage
 
-Before you use the ACL plugin, you need to have properly configured your Service or
+Before you use the ACL plugin, configure your Service or
 Route with an [authentication plugin](/hub/#authentication)
 so that the plugin can identify the client Consumer making the request.
 
-#### Associating Consumers
+<div class="alert alert-warning">
+  <strong>Note:</strong> We support <a href="https://docs.konghq.com/enterprise/2.4.x/support-policy/">compatibility</a> with Kong version 1.5 or greater. 
+</div>
```

Comment:
```suggestion
{:.note}
**Note:** Kong supports [compatibility](https://docs.konghq.com/enterprise/latest/support-policy/) with
{{site.base_gateway}} version 1.5 or greater. 
```
You're going to run into the html tag note styling a lot, as we only just started transitioning away from it. Please use the markdown styling going forward (for more info, see https://docs.konghq.com/contributing/markdown-rules/#admonitions).

I have a question about this note though: was this asked for in the ticket? We don't normally specifically say which gateway versions we support, otherwise we'd have to update that info everywhere every time a support period expires. If someone asked us to add it to the doc, I'd push back on that.

Reply:
This was a swap for an older, outdated note. I can take it out.

Reply:
rm note in commit 037c1e6